src/test/data/unified_full_viewing_keys.json

```diff
@@ -0,0 +1,14 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/unified_full_viewing_keys.py"],
```

Comment:
I verified that these match the test vectors generated from https://github.com/zcash-hackworks/zcash-test-vectors/commit/faed954b99441b5f58b3ad26f8a933c44c97b8fc.